### PR TITLE
Add ldap sql ddl

### LINKED ISSUE #2035: [Optimization][Module Name] modify ldap sql ddl
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar optimization requirement.

### Description

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gaoyan1998**
## Purpose of the pull request
close #2035

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-admin/src/main/resources/db/db-h2.sql [hunk 1]
```diff
@@ -1511,6 +1511,7 @@ CREATE TABLE `dinky_user` (
                             `password` varchar(50) NULL DEFAULT NULL COMMENT 'password',
                             `nickname` varchar(50) NULL DEFAULT NULL COMMENT 'nickname',
                             `worknum` varchar(50) NULL DEFAULT NULL COMMENT 'worknum',
+                            user_type   int    DEFAULT 0 NOT NULL COMMENT 'login type (0:LOCAL,1:LDAP)',
```
Comment (Zzm0809) [inline dinky-admin/src/main/resources/db/db-h2.sql:1514]: UserType 里 定义的是
```java
 LDAP(0, "LDAP"),
    LOCAL(1, "LOCAL");
```

需要对应一下

File: script/sql/dinky.sql [hunk 2]
```diff
@@ -866,6 +866,7 @@ CREATE TABLE `dinky_user`  (
                              `password` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NULL DEFAULT NULL COMMENT 'password',
                              `nickname` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NULL DEFAULT NULL COMMENT 'nickname',
                              `worknum` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NULL DEFAULT NULL COMMENT 'worknum',
+                             `user_type`   int        default 0 not null comment 'login type (0:LOCAL,1:LDAP)',
```
Comment (Zzm0809) [inline script/sql/dinky.sql:869]: UserType 里 定义的是
```java
 LDAP(0, "LDAP"),
    LOCAL(1, "LOCAL");
```

需要对应一下

File: script/sql/upgrade/0.8.0_schema/mysql/dinky_ddl.sql [hunk 3]
```diff
@@ -105,4 +105,7 @@ CREATE TABLE `dinky_git_project` (
 ) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8mb4;
 
 
-ALTER TABLE dinky_role_select_permissions RENAME TO dinky_row_permissions;
\ No newline at end of file
+ALTER TABLE dinky_role_select_permissions RENAME TO dinky_row_permissions;
+
+ALTER TABLE dinky_user
+    add user_type int default 0 not null comment 'login type（0:LOCAL,1:LDAP）' after user_type;
```
Comment (Zzm0809) [inline script/sql/upgrade/0.8.0_schema/mysql/dinky_ddl.sql:111]: UserType 里 定义的是
```java
 LDAP(0, "LDAP"),
    LOCAL(1, "LOCAL");
```

需要对应一下

Comment (gaoyan1998) [reply script/sql/upgrade/0.8.0_schema/mysql/dinky_ddl.sql:111]: ok，UserType修改了， LDAP(1, "LDAP"),
    LOCAL(0, "LOCAL");

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**